docs/vocs/docs/pages/book/advanced-usage/sdk.mdx

```diff
@@ -48,6 +48,16 @@ The following convenience methods are also available to construct a `GenericSdk`
 - `Sdk::standard()` returns a `GenericSdk` supporting all default OpenVM extensions, including RV32IM.
```

Comment:
I think that we should make "VM Configuration" or "Configuring the VM" as a heading at the same level as "Building and Transpiling a Program" and it should come before "Building and Transpiling" because you can't do anything until you're able to construct the `Sdk`. 
Then under this heading, make sub-headings for "Customizing the VM Configuration" and the "Preset Configurations" (perhaps presets should come before customizing?)